pennylane_lightning/core/_serialize.py

```diff
@@ -52,9 +52,12 @@ class QuantumScriptSerializer:
     """
```

Comment:
To keep note, the class docstring is incomplete. It was missing use_mpi and now split_obs.